diagnostic.md

```diff
@@ -3,36 +3,47 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- Ember provides a complete solution when it comes to a client side app.
+- The app runs within the users browser and is self contained, including everything it needs.
+- Ember is meant to be completely backward compatible so that past, or even outdated, innovations
+  are maintained.
```

Comment:
Yes. Three more specific reasons might be:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.